_posts/2017-02-06-the-4-key-elements-of-a-conversation.md

```diff
@@ -0,0 +1,91 @@
+</div>
+
+<p class="text-side-by-side-image" >
+<a href="http://poncho.is/">Poncho</a> is a chatbot that provides weather forecasts. Despite having a conversational interface that is better than most, Poncho lets the users down by not providing good turn-taking. As you can see in screenshot, in the second turn the Poncho has to interrupt the conversation because it is not able to understand the user’s prompt. That is not a good conversation. In this example, the conversation only has one turn. In a real human-to-human conversation that’s similar to meeting a friend and having the following dialog:</p>
+
+<pre>
+Me: “Hi Jane, How are you?”
+Jane: “Hi, I’m good thank you!”
+<emp>Jane walks away...</emp>
+</pre>
+
+<p>
+Now that's an awkward conversation –people rarely have conversations that last a single turn.
+</p>
+<br class="clear-both" />
+
+## 2. Context
+> If the system can handle more than one turn, it should remember what occurred in previous conversational turns (context).
+
+<a href="https://www.google.co.uk/search/about/">Google Voice Search</a> does a good job.
+Asking Google Voice “Who was Oscar Wilde?” gives the user the following result:
+
+<p>
+
+<div class="images-side-by-side-left">
+  <img src="/assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals/google-voice-search-1.png" alt="Hi Poncho on Facebook Messenger" width="250" height="300" />
+</div>
+
+<div class="images-side-by-side-right">
+  <img src="/assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals/google-voice-search-2.png" alt="Hi Poncho on Facebook Messenger" width="250" height="300" />
+</div>
+
+</p>
+
+In the second turn the user responds with, “Can you tell me more?”, a human-like conversation where the name of the author has been omitted, the system understands that they are referring to Oscar Wilde and it will show relevant results.
+
+
+## 3. Reference
+> The system should understand follow up questions that include pronouns, in order to feel more conversational.
+
+<div class="image-side-by-side-text">
+  <img src="/assets/img/custom/blog/2017-02-06-conversational-interface-design-fundamentals/mitzuku-chatbot.jpg" alt="Mitzuku on Facebook Messenger" width="250" height="300" />
+</div>
+
+<p class="text-side-by-side-image">
+<a href="http://www.mitsuku.com/">Mitzuku</a> is a chatbot that won the Loebner Prize in 2013 and 2016. Users can have very long conversations with her, and its site also provides examples of users’ most recent <a href="http://www.square-bear.co.uk/mitsuku/chatlogs.htm">chatlogs</a>. In the screenshot you can see that the system understands pronouns, so that it can refer back to something that was previously mentioned.
+</p>
+
+<br class="clear-both" />
+
+## 4. Variety
+> The system should ask open questions, only if it is able to handle different ways of saying the same thing.
```

Comment:
The system should only ask open questions if it is able to handle different ways of saying the same thing.